src/options.rs

```diff
@@ -122,7 +123,7 @@ impl ConfigOptions {
         let cdir = cargo_dir();
         ConfigOptions {
             crates_file: match matches.value_of("cargo-dir") {
-                Some(dirs) => (dirs.to_string(), fs::canonicalize(dirs).unwrap()),
+                Some(dir) => (format!("{}/.crates.toml", dir), fs::canonicalize(dir).unwrap().join(".crates.toml")),
```

Comment:
*What?* Look at `options.rs`#L36-37. Then at `cargo-install-update.md`#L55-59. Then at `cargo-install-update-config.md`#L45-49. Then at `main.rs`#L33. Then at `ops/mod.rs`#L204-232. And maybe, if you're not a bloody squid, you'll connect the blithering dots on this and the fact that **everything is internally and externally consistent** ***and*** **you broke just about the main composite of the library with this shitty fucking commit** will dawn on you.